schemas/common/response_data.json

```diff
@@ -290,9 +290,9 @@
       "minProperties": 1
     }
   },
-  "feedback_response": {
+  "feedback_response_legacy": {
     "type": "object",
-    "description": "This is used for data_version 0.0.1 or 0.0.3 feedback submissions.",
+    "description": "This is used for data_version 0.0.1 or 0.0.3 feedback submissions by legacy v2 runner..",
```

Comment:
I don't think the "new" legacy feedback was used by v2

Reply:
The "new" legacy is the current one in live hence v2 uses it, unless you meant something different?.

Reply:
I thought the currrent runner was v3 runner?

Reply:
There is no 'v2/v3' anymore, just 1 runner, I think the wording is just misleading and unneeded.

Reply:
Yeah, that makes sense, the wording is incorrect and misleading, updated 👍